cmd/lib.go

```diff
@@ -36,48 +30,58 @@ var defaultStoreHostTargetPath = "/var/lib/resonance"
 
 func wrapHost(ctx context.Context, hst host.Host) (host.Host, error) {
 	var err error
-	if sudo {
+	switch options {
+	case "sudo":
 		hst, err = ihost.NewSudo(ctx, hst)
 		if err != nil {
 			return nil, err
 		}
-	}
-
-	if !disableAgent && ssh != "" {
-		var err error
-		hst, err = ihost.NewAgent(ctx, hst)
-		if err != nil {
-			return nil, err
+	case "sudo,disable-agent", "disable-agent,sudo":
+		if ssh != "" {
+			var err error
+			hst, err = ihost.NewAgent(ctx, hst)
+			if err != nil {
+				return nil, err
+			}
+			hst, err = ihost.NewSudo(ctx, hst)
+			if err != nil {
+				return nil, err
+			}
+		}
+	case "disable-agent":
+		if ssh != "" {
+			var err error
+			hst, err = ihost.NewAgent(ctx, hst)
+			if err != nil {
+				return nil, err
+			}
 		}
+	default:
+		return hst, nil
 	}
 
 	return hst, nil
 }
 
 func addHostFlagsCommon(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(
-		&ssh, "ssh", "", defaultSsh,
+		&ssh, "target-ssh", "s", defaultSsh,
 		"Applies configuration to given hostname using SSH in the format: [<user>[;fingerprint=<host-key fingerprint>]@]<host>[:<port>]",
 	)
 
 	cmd.Flags().StringVarP(
-		&dockerContainer, "docker-container", "", defaultDockerContainer,
-		"Applies configuration to given Docker container name",
+		&docker, "target-docker", "d", defaultDocker,
+		"Applies configuration to given Docker container name \n"+
```

Comment:
I haven't tested Cobra with `\n`, does it properly format things, or it is odd looking?

Reply:
I did no worries :blinks: